<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QMainWindow" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>apt-offline</string>
  </property>
  <property name="windowIcon">
   <iconset resource="image.qrc">
    <normaloff>:/images/icons.png</normaloff>:/images/icons.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>200</height>
       </size>
      </property>
      <property name="title">
       <string>Opsi Pencarian</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Distribusi:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="distribusiComboBox"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Nama paket:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>198</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="cariButton">
           <property name="text">
            <string>Cari</string>
           </property>
           <property name="icon">
            <iconset resource="image.qrc">
             <normaloff>:/images/edit-find.png</normaloff>:/images/edit-find.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="cariPaketLineEdit"/>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="downloadListGroupBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="title">
       <string>Daftar paket-paket yang dibutuhkan</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QWidget" name="buttonWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="lanjutkanButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="image.qrc">
              <normaloff>:/images/media-playback-start.png</normaloff>:/images/media-playback-start.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="hentikanButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="image.qrc">
              <normaloff>:/images/media-playback-pause.png</normaloff>:/images/media-playback-pause.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="batalButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="image.qrc">
              <normaloff>:/images/media-playback-stop.png</normaloff>:/images/media-playback-stop.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>302</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTreeView" name="treeView">
         <property name="showDropIndicator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="itemsExpandable">
          <bool>false</bool>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <property name="expandsOnDoubleClick">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="infoTotalLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Informasi Unduhan (Terunduh/Total):</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="terunduhLabel">
           <property name="text">
            <string>0 B</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>/</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="totalLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>0 B</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="pilihSemuaButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Pilih Semua</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="kosongkanSemuaButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Kosongkan Semua</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="unduhButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Unduh</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>622</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Pengaturan">
    <property name="title">
     <string>&amp;Pengaturan</string>
    </property>
    <addaction name="actionFolderUnduhan"/>
    <addaction name="separator"/>
    <addaction name="actionProxy"/>
    <addaction name="separator"/>
    <addaction name="actionKeluar"/>
   </widget>
   <widget class="QMenu" name="menuUnduhan">
    <property name="title">
     <string>Unduhan</string>
    </property>
    <addaction name="actionJumlahUnduhan"/>
   </widget>
   <widget class="QMenu" name="menuBantuan">
    <property name="title">
     <string>Bantuan</string>
    </property>
    <addaction name="actionTentang"/>
    <addaction name="actionTentang_Qt"/>
   </widget>
   <addaction name="menu_Pengaturan"/>
   <addaction name="menuUnduhan"/>
   <addaction name="menuBantuan"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionRepositori">
   <property name="text">
    <string>Repositori</string>
   </property>
  </action>
  <action name="actionFolderUnduhan">
   <property name="text">
    <string>Lokasi folder unduhan...</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionProxy">
   <property name="text">
    <string>Proxy...</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionJumlahUnduhan">
   <property name="text">
    <string>Jumlah unduhan secara bersamaan...</string>
   </property>
  </action>
  <action name="actionKeluar">
   <property name="text">
    <string>Keluar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionTentang">
   <property name="text">
    <string>Tentang</string>
   </property>
  </action>
  <action name="actionTentang_Qt">
   <property name="text">
    <string>Tentang Qt</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>distribusiComboBox</tabstop>
  <tabstop>cariPaketLineEdit</tabstop>
  <tabstop>cariButton</tabstop>
  <tabstop>unduhButton</tabstop>
 </tabstops>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionKeluar</sender>
   <signal>triggered()</signal>
   <receiver>Form</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
